game/scripts/vscripts/Sections/Chapter1/SectionCasting.ts

```diff
@@ -41,8 +49,26 @@ const start = (complete: () => void) => {
         tg.audioDialog(LocalizationKey.Script_1_BreatheFire_3, LocalizationKey.Script_1_BreatheFire_3, ctx => ctx[CustomNpcKeys.SlacksMudGolem]),
 
         tg.immediate(_ => freezePlayerHero(false)),
+        tg.immediate(ctx => listenerID = ListenToGameEvent("dota_player_used_ability", (event: DotaPlayerUsedAbilityEvent) => {
+            if (event.abilityname === abilNameBreatheFire) {
+                eventTimer = Timers.CreateTimer(2, () => {
```

Comment:
I think 2 seconds feels slightly too long after playing it, maybe 1 second?

Reply:
Lowered to 1.25 per our discussion.